backend/api/image_generation.py

```
@@ -213,11 +228,25 @@ async def _generate_persona_avatar_unsafe(persona_name: str, persona_role: str,
-                                        return image_url
+                                        debug_log(f"[FREEPIK] Temporary URL: {temp_image_url}")
+                                        
+                                        # Upload to Wasabi if persona_id is provided
+                                        if persona_id:
+                                            try:
+                                                wasabi_url = await upload_persona_avatar_from_url(persona_id, temp_image_url)
+                                                if wasabi_url and wasabi_url.strip():
+                                                    debug_log(f"[FREEPIK] Uploaded to Wasabi: {wasabi_url}")
+                                                    return wasabi_url
+                                                else:
+                                                    debug_log(f"[FREEPIK] Wasabi upload failed, returning temporary URL")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)